source/install/desktop-app-changelog.rst

```diff
@@ -15,7 +15,15 @@ Release v4.7
 
 **Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
 
-**Released 2021-06-23**
+- **v4.7.1, release ?**
+ - Security fix.
```

Comment:
What is the severity level on this fix https://github.com/mattermost/desktop/pull/1667?

Reply:
Its a P2 Medium severity